From 17bdded69f72f05613c4da6261083632b1646826 Mon Sep 17 00:00:00 2001
From: danie007 <danilselvan@gmail.com>
Date: Wed, 11 Nov 2020 18:20:53 +0530
Subject: [PATCH] Enable secure boot support

- HAB status logs retrievement through the hab_status command
- Support to extend the root of trust

Signed-off-by: danie007 <danilselvan@gmail.com>
---
 configs/mx6ul_14x14_evk_optee_defconfig | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/configs/mx6ul_14x14_evk_optee_defconfig b/configs/mx6ul_14x14_evk_optee_defconfig
index 0f14bd370b..471d33fcd0 100644
--- a/configs/mx6ul_14x14_evk_optee_defconfig
+++ b/configs/mx6ul_14x14_evk_optee_defconfig
@@ -77,3 +77,7 @@ CONFIG_FASTBOOT_BUF_ADDR=0x83800000
 CONFIG_FASTBOOT_BUF_SIZE=0x40000000
 CONFIG_FASTBOOT_FLASH=y
 CONFIG_EFI_PARTITION=y
+
+# Added by Yocto to support the HAB features
+# (Secure boot)
+CONFIG_SECURE_BOOT=y
